api/envoy/config/wasm/v2/wasm.proto

```diff
@@ -39,4 +39,6 @@ message WasmConfig {
   // If true, create a single VM rather than creating one VM per silo. Such a singleton can
   // not be used with filters.
   bool singleton = 5;
+  // If set add 'stats_prefix' as a prefix to all stats.
+  string stats_prefix = 6;
```

Comment:
Nit: everything else in Envoy uses `stat_prefix`.

Also, usually it's mandatory (`[(validate.rules).string.min_bytes = 1];`), though I never liked that part.

Reply:
Changed the name.

Not having a prefix is a specific feature request, so leaving out the min_bytes.